src/blockes/prod-card.scss

```diff
@@ -0,0 +1,132 @@
+
+html {
+  font-family: Roboto, sans-serif;
+  color: $main-accent;
+}
+
+p {
+  margin: 0;
+}
+
+.product-card {
+  display: flex;
+  width: 200px;
+  box-sizing: border-box;
+  flex-direction: column;
+  border: 1px solid white;
+
+  &__imac-image {
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+    margin-top: 32px;
+    align-self: center;
+  }
+
+  &__product-description {
+    color:$main-accent;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin-bottom: 4px;
+    margin-right: $main-shift;
+    margin-left: $main-shift;
+  }
+
+  &__product-code {
+    font-size: 10px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 14px;
+    align-self: start;
+    margin-left: $main-shift;
+    margin-bottom: $main-shift;
+    color: $secondary;
+  }
+
+  &__feadback {
+    display: flex;
+    margin-inline: $main-shift;
+    align-items: end;
+    justify-content: space-between;
+    margin-bottom: 24px;
+  }
+
+  &__reviews-text {
+    font-size: 10px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 14px;
+  }
+
+  &__price {
+    display: flex;
+    margin-inline: $main-shift;
+    justify-content: space-between;
+    margin-bottom: $main-shift;
+  }
+
+  &__price-text {
+    font-size: 12px;
+    font-weight: 400;
+    line-height: 18px;
+    color: $secondary;
+    }
+
+  &__price-amount {
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+    }
+
+  &__button-buy {
+    width: 166px;
```

Comment:
```suggestion
    width: 100%;
```

Reply:
I cant make it because button will be bigger then expected

Reply:
Consider using paddings for the card to avoid it